test/satellite_test.exs

```diff
@@ -30,12 +30,7 @@ defmodule SatelliteTest do
     positionEci = positionAndVelocity.position
     velocityEci = positionAndVelocity.velocity
 
-
-    deg2rad = :math.pi/180
-    observerGd = %{
-       longitude: -122.0308 * deg2rad,
-       latitude: 36.9613422 * deg2rad,
-       height: 0.370}
+    observerGd = Observer.create_from(36.9613422, -122.0308, 0.370)
```

Comment:
Use new helper method. Still not sure if i like how this looks, since latitude and longitude are easy to mix up with this new method. It was more explicit before.